src/theme/CodeBlock/styles.module.css

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
if this file contains the customizations you implemented, I'd suggest to make those customization changes in `src/css/custom.css` file instead, this addition is probably unnecessary.

Reply:
(or perhaps `src/css/unity-custom.scss` if you'd like to have SASS support etc.)

Reply:
I can try adding them to the global, but sometimes a swizzle is required to affect global. These css states are generated with different names on build, which causes the global to be erratic on a swizzle. Swizzles it's best to keep the css in the swizzled css file.

Reply:
please at least give it a try so that we wouldn't need to maintain those 2 files (js and css) added for CodeBlock stuff. also, if a CSS property doesn't apply, it might worth trying `!important` at the end of property value like `font-size: 14px !important;`.
having said that, if these `src/css/*` files doesn't work even after `!important` annotation, maybe it is really required to do swizzling — I'm just pointing the fact that we didn't try these options yet :)